pkg/controller/humiocluster/defaults.go

```diff
@@ -113,48 +117,60 @@ func humioServiceAccountNameOrDefault(hc *humioClusterv1alpha1.HumioCluster) str
 	if hc.Spec.HumioServiceAccountName != "" {
 		return hc.Spec.HumioServiceAccountName
 	}
-	return humioServiceAccountName
+	return fmt.Sprintf("%s-%s-%s", hc.Namespace, hc.Name, humioServiceAccountNameSuffix)
```

Comment:
For all of these, is there a reason we include the namespace in the name of the resource? I think all of these resources are already namespaced.

Reply:
Right! That was actually my initial plan. Doh! I will update the PR. Thanks!